homeassistant/components/openuv/__init__.py

```diff
@@ -47,6 +51,36 @@
 
 PLATFORMS = [Platform.BINARY_SENSOR, Platform.SENSOR]
 
+SERVICE_NAME_UPDATE_DATA = "update_data"
+SERVICE_NAME_UPDATE_PROTECTION_DATA = "update_protection_data"
+SERVICE_NAME_UPDATE_UV_INDEX_DATA = "update_uv_index_data"
+
+SERVICES = (
```

Comment:
Why are these services necessary. Can a user not use the `homeassistant.update_entity` service for this?

Reply:
Good question! I don't know if we had the entity update service when this was first merged, but nevertheless, we can absolutely go that way.

Bonus: that path will allow me to turn this from a breaking change into a deprecation with a Repairs item.